README.md

```diff
@@ -19,14 +19,27 @@ Libraries that produce telemetry data should only depend on `api`
 and defer the choice of the SDK to the application developer. Applications may
 depend on `sdk` or another package that implements the API.
 
-To install the API and SDK packages,
+All packages are published to [https://pkg.go.dev/go.opentelemetry.io/otel](https://pkg.go.dev/go.opentelemetry.io/otel) and is the preferred location to import from.
 
+## Building the code
```

Comment:
I think that this basically duplicates what's written in CONTRIBUTING.md, so instead of having `## Building the code` section, I'd move the `## Contributing` section from below to here. And then modify the CONTRIBUTING.md file to remove the `go get` invocation.

Reply:
+1 for not adding this `Building the code` since we already have the `Contributing` and it is harder to maintain two files that have similar content.